### DISCUSSION TITLE
Suggestion for enrichment analysis for small microarray set.

### DISCUSSION THREAD

**Discussion by zhen-fu** [YuLab-SMU/clusterProfiler#599] [Q&A]
I am working on a microarray dataset that has 860 genes in total. I did DE gene expression analysis, there were 60 up regulated genes and 80 down-regulated genes.  

I would like to do enrichment analysis. Would it appropriate to use  over-representation analysis (ORA), or even gene set enrichment analysis (GSEA)? My concern is the 860 gene microarray is a much reduced gene set compared to whole transcriptome, and it might introduce bias. Any comments are welcome!

**Comment by huerqiang** [comment]
In this case, the ORA algorithm is suitable.
